test/fixtures/grid/col-xl-0.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Test</title>
+        <!--[if lt IE 9]>
```

Comment:
not usefull on BS4

Reply:
True (see twbs/bootstrap#21476) but not relevant for this PR. IMHO, we should track such little details in some issue and then do it all at once at the end.

Reply:
#421